src/ventures-for-shared-prosperity/index.njk

```diff
@@ -7,63 +7,59 @@ title: Ventures for Shared Prosperity
+      <a href="/apply">
+        Apply to be a member of our first cohort of entrepreneurs.
+      </a>
     </p>
-    <p>
-      <a href="/apply" class="btn hero__btn">Apply to be a founder</a>
+
+    <p class="hero__body g-max-3-half g-centered">
+      Not sure you are ready to apply? Learn more about what we provide our entrepreneurs, here, how we think about building scalable, impactful businesses here, and why we think tackling the problem of the excess costs of poverty is an important one. At Ventures for Shared Prosperity we believe we can help improve the lives of individuals living in poverty by partnering with talented entrepreneurs to build and scale software powered, behavioral science informed businesses that address some of the diverse, complex and structural causes of poverty in America.
     </p>
+
+    <div>
+      <img src="" alt="">
+      <img src="" alt="">
+      <img src="" alt="">
+      <h3>Join a Q&A with the team </h3>
+      <p>Learn more about the ideas42 Venture Studio and our business building program at one of the following open ended conference calls.</p>
+      <ul>
+        <li>
+          <time>October 1, 2020 at 11:00 am EST </time>
+        </li>
+        <li>
+          <time>October 3, 2020 at 5:00 pm EST</time>
+        </li>
+        <li>
+          <time>October 8, 2020 at 11:00 am EST</time>
+        </li>
+        <li>
+          <time>October 10, 2020 at 5:00 pm EST</time>
+        </li>
+      </ul>
+    </div>
+  
   </div>
-  <img class="masthead__graphic masthead__graphic--centered" src="/images/masthead-graphic-triangle-subtle.svg" alt="" />
+  
+ 
 {% endblock %}
 
 {% block main %}
 <div class="standard-restricted-width">
-  <header class="content-block__intro">
-    <h2>Mission</h2>
-    <p>
-      We need this gig! We're on a mission from God.
-    </p>
-  </header>
-
-  <header class="content-block__intro g-max-3">
-    <h2>What we provide</h2>
-    <p>
-      We don’t have all the answers and we won’t have it all figured out when you 
-      join us. But we know it takes a solid foundation to build a thriving technology 
-      business. We provide that foundation.
-    </p>
-  </header>
-
-  <ul class="home-benefits__list g-max-3">
-    <li>
-      <h3>Compensation</h3>
-      <p>
-        We pay a $75k annual salary and provide health benefits
-        so you can focus on the work.
-      </p>
-    </li>
-    <li>
-      <h3>Expertise</h3>
-      <p>
-        We surround you with a team of seasoned experts so you can build
-        world-class digital products.
-      </p>
-    </li>
-    <li>
-      <h3>Energy</h3>
-      <p>
-        We lift you up with encouragement, feedback, and kindness so you can bring
-        your best self to the process.
-      </p>
-    </li>
-  </ul>
+<h3>Join our first cohort of entrepreneurs.</h3>
```

Comment:
You can remove this Apply CTA content from this page. This element–the yellow section from the mocks–will be a globally shared element so we don't need to repeat it here.

Reply:
Fixed!